src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -5,32 +5,50 @@
 import nextstep.subway.line.domain.LineRepository;
 import nextstep.subway.line.dto.LineRequest;
 import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.section.domain.Section;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.domain.StationRepository;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 import java.util.stream.Collectors;
 
 import static nextstep.subway.exception.ApiExceptionMessge.NOT_FOUND_LINE;
+import static nextstep.subway.exception.ApiExceptionMessge.NOT_FOUND_STATION;
 
 @Service
 @Transactional
 public class LineService {
-    private final LineRepository lineRepository;
+    private final LineRepository lines;
+    private final StationRepository stations;
 
-    public LineService(LineRepository lineRepository) {
-        this.lineRepository = lineRepository;
+    public LineService(final LineRepository lineRepository, final StationRepository stationRepository) {
+        this.lines = lineRepository;
+        this.stations = stationRepository;
     }
 
-    @Transactional
     public LineResponse saveLine(final LineRequest request) {
-        Line persistLine = lineRepository.save(request.toLine());
+        Line persistLine = lines.save(makeLine(request));
         return LineResponse.of(persistLine);
     }
 
+    private Line makeLine(final LineRequest request) {
```

Comment:
```suggestion
    private Line create(final LineRequest request) {
```
보통 객체 생성은 create으로 씁니다.

Reply:
createLine으로 변경했습니다! ( 791bc22 )